tests/test_tasker.py

```diff
@@ -15,13 +15,16 @@
 from requests.packages.urllib3.exceptions import ProtocolError
 from tests.util import requires_internet
 
+import os
+import json
 import docker
 import docker.errors
 import requests
 import sys
 import time
 import atomic_reactor
 from docker.errors import APIError
+from base64 import b64encode
```

Comment:
I know that we don't follow PEP8 module imports guidelines but I cannot resist :). This import should be first (and alphabetical order json before os)

```
# import stdlib modules

# import 3rd party modules

# import project modules
```

https://www.python.org/dev/peps/pep-0008/#imports